ts/features/fci/store/reducers/fciSignatureFieldDrawing.ts

```diff
@@ -0,0 +1,97 @@
+import * as O from "fp-ts/lib/Option";
+import { getType } from "typesafe-actions";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { pipe } from "fp-ts/lib/function";
+import { Action } from "../../../../store/actions/types";
+import { GlobalState } from "../../../../store/reducers/types";
+import { fciDocumentSignatureFields } from "../actions";
+
+/**
+ * The drawn document type used in a pot.
+ */
+export type DrawnDocument = {
+  document: string;
+  page: number;
+};
```

Comment:
What do you think about using a pot in type?

```suggestion
export type DrawnDocument = pot.Pot<{
  document: string;
  page: number;
}, Error>;
```